restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/api/incoming-phone-numbers-api.adoc

```diff
@@ -578,3 +578,42 @@ match different numbers.
 
 
 
+[[resource-uri-5]]
+==== Resource URI
+
+*/2012-04-24/Accounts/\{TargetAccountSid}/IncomingPhoneNumbers/migrate*
```

Comment:
as in accoutn migration API, should we docuemnt possible response status? not sure this is current practice, but i expect status codes to be explained...

Reply:
currently adding status codes is not part of practice since mostly we use standard http codes not restcomm specific.